config/routes.rb

```diff
@@ -465,6 +453,18 @@
     end
   end
 
+  resources :groups do
+    member do
+      # this should be put into it's own resource
+      match '/members' => 'groups#add_users', via: :post, as: 'members_of'
+      match '/members/:user_id' => 'groups#remove_user', via: :delete, as: 'member_of'
+      # this should be put into it's own resource
+      match '/memberships/:membership_id' => 'groups#edit_membership', via: :put, as: 'membership_of'
+      match '/memberships/:membership_id' => 'groups#destroy_membership', via: :delete
+      match '/memberships' => 'groups#create_memberships', via: :post, as: 'memberships_of'
+    end
+  end
```

Comment:
Can we get the show page to be outside of the admin scope but the rest to stay inside it? How much of a hassle would that be?

Reply:
That's easily doable with two `resources` blocks. Will do